raiden/tasks.py

```diff
@@ -1,17 +1,33 @@
 import time
-import structlog
+import threading
+from pkg_resources import parse_version, require
 
 import gevent
 from gevent.event import AsyncResult
-from gevent.queue import (
-    Queue,
-)
+from gevent.queue import Queue
+import structlog
+
 from raiden.exceptions import RaidenShuttingDown
 
+CHECK_VERSION_INTERVAL = 60 * 5
 REMOVE_CALLBACK = object()
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+def check_version():
+    """Check every 3 hours for a new release"""
+    app_version = parse_version(require('raiden')[0].version)
```

Comment:
Please use `get_system_spec()['raiden']` as the input to `parse_version()` here to be consistent with the other places the version is used.

Reply:
I've tried to use it, it did not work as it should.

Reply:
Can you describe what problem you ran into?